packages/react-admin/src/sideEffect/saga/crudResponse.js

```diff
@@ -39,6 +45,17 @@ function* handleResponse({ type, requestPayload, error, payload }) {
                       ),
                   ])
                 : yield [put(showNotification('ra.notification.updated'))];
+        case CRUD_UPDATE_MANY_SUCCESS: {
+            const actions = [put(showNotification('ra.notification.updated'))];
```

Comment:
I think we need a custom message here, something like `3 elements deleted`, instead of `'Element deleted`. Same for update down there.

Reply:
Also, because this is a batch request, the response can contain success and failed responses.So this should be separated (like I did in my PR). 
I think the failed response should keep selected state (so a retry is easy) and a individual notification should be shown for each failed entity. 
A general message like `3 elements deleted, 2 failed`